kind-demo-super-heroes/resources-kubernetes.yml

```diff
@@ -0,0 +1,1768 @@
+#######################################################################
+# THIS FILE IS AUTOMATICALLY GENERATED DURING CI/CD.
+# ANY LOCAL CHANGES YOU MAKE SHOULD NOT BE COMMITTED TO SOURCE CONTROL.
```

Comment:
????

Reply:
I copied this file from the super-heroes repo and update it for the demo. Gonna remove this part. Thanks!